### PR TITLE
Test authentication controller

### LINKED ISSUE #36: Write tests for the AuthenticationController
Request tests for the controller

### PR DESCRIPTION
**Pull request by redferret**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Rubocop has passed locally and any fixes were made for failures
- [ ] CircleCI build is passing
- [x] All tests passing locally

## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): Adds more test coverage

## What is the current behavior?

Issue Number: 
closes #36

## What is the new behavior?

- When the endpoint `/token_auth` is requested by a client with an invalid token, it will return a proper server response.

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by Jaybraum — APPROVED** [review]
looks good!

File: app/controllers/authentication_controller.rb [hunk 1]
```diff
@@ -2,13 +2,17 @@ class AuthenticationController < ApplicationController
...
+        user.save
+        render json: { is_new: false, token: token }
+      else
+        User.create!(name: payload[:name], email: payload[:email], google_token: token)
+        render json: { is_new: true, token: token }
+      end
     else
-      User.create!(name: payload[:name], email: payload[:email], google_token: token)
-      render json: { is_new: true, token: token }
+      render json: payload, status: :bad_request
```
Comment (mkrumholz) [inline app/controllers/authentication_controller.rb:15]: Nice! Way to add a sad path in here.

**Review by mkrumholz — APPROVED** [review]
All passing now with some updated cassettes. Looks good to me!

**Review by mkrumholz — CHANGES_REQUESTED** [review]
Hold up one sec -- I made a silly error on joining

**Review by mkrumholz — APPROVED** [review]

**Pull request merged**